ibm/mas_devops/roles/suite_db2_setup_for_manage/tasks/main.yml

```diff
@@ -77,13 +77,20 @@
 - name: Run setupdb to prepare Db2 for use with Maximo TPAE
```

Comment:
@sachbalag be sure to include in PR's.title (we use to ad in any commit messages) the semantic version prefix in squared brackets... in this case it seems a `[minor]` to me as we slightly changed behavior